Code/UnitTests/FoundationTest/SimdMath/SimdTransformTest.cpp

```diff
@@ -38,6 +38,29 @@ EZ_CREATE_SIMPLE_TEST(SimdMath, SimdTransform)
       EZ_TEST_BOOL((t.m_Scale == ezSimdVec4f(1)).AllSet<3>());
     }
 
+    {
+      ezSimdQuat qRot;
```

Comment:
Use new MakeFromAxisAndAngle function

Reply:
All these changes come in a separate PR.